pandas/conftest.py

```diff
@@ -675,3 +675,20 @@ def tick_classes(request):
         normalize=st.booleans(),
         startingMonth=st.integers(min_value=1, max_value=12)
     ))
+
+lorem_ipsum_text = (
+    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod"
+    " tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim"
+    " veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex"
+    " ea commodo consequat. Duis aute irure dolor in reprehenderit in"
+    " voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur"
+    " sint occaecat cupidatat non proident, sunt in culpa qui officia"
+    " deserunt mollit anim id est laborum.")
+
+
+@pytest.fixture(scope='session')
```

Comment:
Do you need to explicitly state scope? Assumed this to be session given location but maybe have that wrong

Reply:
the scope parameter of the fixture is the scope for which this fixture is shared. the default is to execute the fixture function and all of the dependent fixtures for each parametrized run of each test.
the session scope here just means that the fixture function is only executed once and reused each time. here the fixture setup is trivial so specifying the scope probably doesn't achieve anything.